doc/getting-started/getting-started/installation.md

```diff
@@ -26,15 +26,15 @@ sudo tar -xvf <release>.tar.xz -C /usr/local --strip-components=2
 You will need some Python packages if you want to use clnrest.  Unfortunately there are some Python packages which are not packaged in Ubuntu, and so you will need to force installation of these (I recommend --user which will install them in your own .local directory, so at least you won't run the risk of breaking Python globally!).
 
 ```
-sudo apt-get install python3-json5 python3-flask python3-gunicorn
-pip3 install --user flask_restx pyln-client
+sudo apt-get install python3-flask python3-gunicorn
+pip3 install --user flask_restx pyln-client flask-socketio gevent gevent-websocket
```

Comment:
Ah you fixed something that I was struggling in https://github.com/ElementsProject/lightning/pull/6739

Thanks for fixing the python dependencies

Reply:
Cool!